logicrunner/builtin/contract/member/member.go

```diff
@@ -142,7 +142,11 @@ func (m *Member) Call(signedRequest []byte) (interface{}, error) {
 		return m.memberGet(request.Params.PublicKey)
 	}
 
-	params := request.Params.CallParams.(map[string]interface{})
+	var params map[string]interface{}
+	var ok bool
+	if params, ok = request.Params.CallParams.(map[string]interface{}); !ok {
+		return nil, fmt.Errorf("failed to cast request.Params.CallParams: expected map[string]interface{}, got %T", request.Params.CallParams)
```

Comment:
This error will be returned to client. Should be discussed with @sofiayem

Reply:
Is exists standard error for incorrect input params?

Reply:
Not yet